<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainPage</class>
 <widget class="QWidget" name="mainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>212</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>212</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>212</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DatasetTool</string>
  </property>
  <widget class="QPushButton" name="pbCountBalance">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Calcular Caja</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdTotalBalance">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMonthFolder">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSetMonthFolder">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Carpeta Mes</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>256</width>
     <height>190</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
